samples/Cpp/XAMLBehaviorsSampleCpp/XAMLBehaviorsSampleCpp.vcxproj

```diff
@@ -42,37 +50,49 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v140</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```

Comment:
I don't think it matters for the sample, right? ARM64 is supported on v141+.

Reply:
Oh, you mean you think that this is the reason the CI failed?

Reply:
No, that hadn't occurred to me. I was worried that some folks might need to install a different sdk for v142 if using VS2017, and since we're using v141 elsewhere it would make sense to keep them the same.

Do you have any idea why the CI failed?

Reply:
This makes sense. I switched back to 141. Let me wait for the CI to run again to see what is happening. I think it might be just an old version of the build server itself.